src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -63,49 +59,13 @@ public MainWindow(Stage primaryStage, Logic logic) {
         // Configure the UI
         setWindowDefaultSize(logic.getGuiSettings());
 
-        setAccelerators();
-
         helpWindow = new HelpWindow();
     }
 
     public Stage getPrimaryStage() {
         return primaryStage;
     }
 
-    private void setAccelerators() {
-        setAccelerator(helpMenuItem, KeyCombination.valueOf("F1"));
-    }
-
-    /**
-     * Sets the accelerator of a MenuItem.
-     * @param keyCombination the KeyCombination value of the accelerator
-     */
-    private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
-        menuItem.setAccelerator(keyCombination);
-
-        /*
-         * TODO: the code below can be removed once the bug reported here
-         * https://bugs.openjdk.java.net/browse/JDK-8131666
-         * is fixed in later version of SDK.
-         *
-         * According to the bug report, TextInputControl (TextField, TextArea) will
-         * consume function-key events. Because CommandBox contains a TextField, and
-         * ResultDisplay contains a TextArea, thus some accelerators (e.g F1) will
-         * not work when the focus is in them because the key event is consumed by
-         * the TextInputControl(s).
-         *
-         * For now, we add following event filter to capture such key events and open
-         * help window purposely so to support accelerators even when focus is
-         * in CommandBox or ResultDisplay.
-         */
-        getRoot().addEventFilter(KeyEvent.KEY_PRESSED, event -> {
-            if (event.getTarget() instanceof TextInputControl && keyCombination.match(event)) {
-                menuItem.getOnAction().handle(new ActionEvent());
-                event.consume();
-            }
-        });
-    }
-
```

Comment:
hmm, why was this removed?

Reply:
If anything, I think it's only line 86-106 that should be removed.

Reply:
Yee it wasn't compatible with the new menu navbar I've implemented, I'll include the F1 hotkey again in a future commit.

Reply:
oh ok that's not a big problem

Reply:
@malwaregarry I've added back the F1 hotkey in a new function `setHelpShortcut` in the latest commit, don't think we need multiple layers of abstraction (i.e. `setAccelerators()` calling `setAccelerator(helpMenuItem, KeyCombination.valueOf("F1"));` in the old code) for now, unless we want to add more shortcuts in the future.